app/calculate_chart.py

```diff
@@ -111,35 +115,38 @@ def f(t):
 
 	alph = 0.6
 	plt.plot(temperature, saturation, 'k-', linewidth = 1, alpha = 1)
-	plt.plot(temperature, saturation*.9, 'k-', linewidth = 1 , alpha = 1)
+	plt.plot(temperature, saturation*.9, 'k-', linewidth = 1 , alpha = alph)
 	plt.plot(temperature, saturation*.8, 'k-', linewidth = 1, alpha = alph)
 	plt.plot(temperature, saturation*.7, 'k-', linewidth = 1,alpha = alph)
 	plt.plot(temperature, saturation*.6, 'k-', linewidth = 1,alpha = alph)
 	plt.plot(temperature, saturation*.5, 'k-', linewidth = 1,alpha = alph)
 	plt.plot(temperature, saturation*.4, 'k-', linewidth = 1,alpha = alph)
 	plt.plot(temperature, saturation*.3, 'k-', linewidth = 1,alpha = alph)
 	plt.plot(temperature, saturation*.2, 'k-', linewidth = 1,alpha = alph)
-	plt.plot(temperature, saturation*.1, 'k-', linewidth = 1,alpha = 1)
-
-	#Forced Convection Plots. 
-	levels_psy_forced = np.linspace(np.amin(T_MRT_forced_psy), np.amax(T_MRT_forced_psy), 15)
-	levels_wb_forced = np.linspace(-0.01,0.01,2)
-	CS3=plt.contourf(Y, psy_sat, T_MRT_forced_psy, cmap = 'jet', levels=levels_psy_forced,interpolation='sinc', fontsize = 20, dpi = 1200, alpha = 1)
-	CS = plt.contour(Y, psy_sat, T_MRT_forced_psy, 20, colors='k', alpha = 1)
-	plt.clabel(CS, inline=3, fmt='%1.1f', fontsize=textsize)
-	#plt.contour(Y, psy_sat, wet_bulb_compare_forced, cmap = 'ocean', levels=levels_wb_forced, interpolation='sinc', fontsize = 20, dpi = 1200)
-	
-	#PJ commented these out
-	#plt.contour(Y, psy_sat, equivalent_forced, cmap = 'ocean', levels=levels_wb_forced, interpolation='sinc', fontsize = 20, dpi = 600)
-	#plt.contour(Y, psy_sat, dew_point_compare_forced, cmap = 'ocean', levels=levels_wb_forced, interpolation='sinc', fontsize = 20, dpi = 600)
-	#plt.contourf(Y, psy_sat, Q_comp_rat, cmap = 'ocean', levels=levels_one, interpolation='sinc', fontsize = 20, dpi = 600)
-	#plt.contourf(Y, psy_sat, Q_comp_rat, cmap = 'ocean', levels=levels_half, interpolation='sinc', fontsize = 20, dpi = 600)
-	#plt.contourf(Y, psy_sat, Q_comp_rat, cmap = 'ocean', levels=levels_two, interpolation='sinc', fontsize = 20, dpi = 600)
-	#plt.contourf(Y, psy_sat, Q_comp_rat, cmap = 'ocean', levels=levels_three, interpolation='sinc', fontsize = 20, dpi = 600)
-	#plt.contourf(Y, psy_sat, Q_comp_rat, cmap = 'ocean', levels=levels_four, interpolation='sinc', fontsize = 20, dpi = 600)
-	#plt.contourf(Y, psy_sat, Q_comp_rat, cmap = 'ocean', levels=levels_six, interpolation='sinc', fontsize = 20, dpi = 600)
-	cbar = plt.colorbar(CS3, orientation='vertical', format="%.1f")
+	plt.plot(temperature, saturation*.1, 'k-', linewidth = 1,alpha = alph)
+
+	if dep != 0:
```

Comment:
When `dep!=0` the countour plot doesn't plot. I just get a blank graph

Reply:
![download](https://user-images.githubusercontent.com/12176132/58110407-75fc1180-7bbd-11e9-9141-a73d6a4a1512.png)

that's what i get for 

![Screen Shot 2019-05-21 at 11 42 23 AM](https://user-images.githubusercontent.com/12176132/58110449-8c09d200-7bbd-11e9-9f12-5ace952af606.png)

im not sure why this would be. is there not a colorbar either for you? does this happen? plt.text(38.5,-1.75,"m/s", size=textsize) (line 141 in calculate_chart.py)

Reply:
I think i was putting random large values and throwing the graph off the chart.. perhaps we can add some upper bounds in the inputs in a future update